src/index.js

```diff
@@ -0,0 +1,244 @@
+import fetch from 'node-fetch';
+import {
+  DynamoDBClient,
+} from '@aws-sdk/client-dynamodb';
+import {
+  DynamoDBDocument,
+} from '@aws-sdk/lib-dynamodb';
+import {
+  GetSecretValueCommand,
+  SecretsManagerClient,
+} from '@aws-sdk/client-secrets-manager';
+
+const REGION = process.env.AWS_REGION;
+const ROTATIONS_TABLE = process.env.ROTATIONS_TABLE;
+const SLACK_TOKEN_SECRET_NAME = process.env.SLACK_TOKEN_SECRET_NAME;
+
+class Member {
+  constructor(
+      victorOpsUserId,
+      slackUserId,
+  ) {
+    this.victorOpsUserId = victorOpsUserId;
+    this.slackUserId = slackUserId;
+  }
+}
+
+class Record {
+  constructor(
+      victorOpsGroupId,
+      slackUserGroupId,
+      members,
+  ) {
+    this.victorOpsGroupId = victorOpsGroupId;
+    this.slackUserGroupId = slackUserGroupId;
+    this.members = members;
+  }
```

Comment:
```suggestion
  /**
  * @param {string|number} victorOptsGroupId
  * @param {string|number} slackUserGroupId
  * @param {Member[]} members
  **/
  constructor(
      victorOpsGroupId,
      slackUserGroupId,
      members,
  ) {
    this.victorOpsGroupId = victorOpsGroupId;
    this.slackUserGroupId = slackUserGroupId;
    this.members = members;
  }
```

I'm only going to comment this function with type hints but by using [jsdoc](https://jsdoc.app) to provide type hints you can help anyone making future changes to get better auto completion.

Reply:
This is great, I've updated to include this.  One piece I still need to figure out is how to properly link to imported pieces such as https://github.com/elbe0046/rotation-updater/commit/1846e57efc9fc73fe0eec3cea426cfb1dff794cf#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R18.

Reply:
Those I believe should be resolving based on what is in your `node_modules` directory. You may need to configure your LSP server to know what types it should be aware of. In theory it should already be resolving [this d.ts file](https://github.com/aws/aws-sdk-js/blob/master/clients/dynamodb.d.ts) but you may need to update your configuration